### ISSUE TITLE
Generate default configuration depending on the component file

### ISSUE THREAD

**Issue by SheepFromHeaven** [biotope/biotope-preview#47]
I want to be able to have my component written and generate a config file from the ts file.

**Comment by no-creative-name** [comment]
@SheepFromHeaven Which ts file do you mean? The components source code ts?

**Comment by SheepFromHeaven** [comment]
Yes, basically looking at the attributes configuration of the biotope element

**Comment by no-creative-name** [comment]
I like this idea.

**Comment by no-creative-name** [comment]
**Concept**

The user doesn't actually have to create a config file for the preview. The source files of his components already contain most of the required information, therefore there is much redundance with the additional preview configs. Features like knobs and different component configurations would need more information than there are in the source files as of now, which means the interfaces for biotope elements may need to be extended.
